app/views/shared/_articles_list.html.erb

```diff
@@ -0,0 +1,44 @@
+<% @articles.each do |article| %>
+  <article class="article">
+    <% if @show_article_author %>
+      <header class="article-name-container">
+        <a href="<%= articles_list_path(article.user.id) %>">
+        <p>
+          <%= article.user.name %>
+        </p>
+        <p>
+          <%= article.user.surname %>
+        </p>
+        </a>
+      </header>
+    <% end %>
+    <section>
+      <%= article.message %>
+    </section>
+    <footer>
+      <div>
+        Posted
+        <time datetime="<%= article.created_at %>">
+          <%= time_ago_in_words(article.created_at) %>
+        </time>
+        ago
+      </div>
+      <% if article.user.id == current_user.id %>
+        <div>
+          <a
+            href="<%= edit_article_path(article) %>"
+          >
+            Edit
+          </a>
+          <button
+            data-id="<%= article.id %>"
+            data-action="remove"
+          >
+            Remove
+          </button>
+        </div>
+      <% end %>
+    </footer>
+  </article>
+<% end %>
+<%= javascript_include_tag "articles.js" %>
```

Comment:
I think you can put it in `content_for :head` call because you have a DOMContentLoaded listener there